sdk/storage/Azure.Storage.Blobs/README.md

```diff
@@ -188,6 +188,8 @@ Get started with our [Blob samples][samples]:
 1. [Hello World](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/samples/Sample01a_HelloWorld.cs): Upload, download, and list blobs (or [asynchronously](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/samples/Sample01b_HelloWorldAsync.cs))
 2. [Auth](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/samples/Sample02_Auth.cs): Authenticate with connection strings, public access, shared keys, shared access signatures, and Azure Active Directory.
 
+
```

Comment:
Can we also add samples using the upload/download directory extension methods in this readme?

Reply:
Wouldn't that be misleading? This README since this is the Azure.Storage.Blobs package README? they would have to download the Azure.Storage.DataMovement and Azure.Storage.DataMovement.Blobs package. We should just keep the samples in the blobs package. It's kinda why I added this in the Next Steps for more advanced scenarios.

Reply:
Actually let me write something that prefaces that you have to download these packages beforehand, hopefully that points people in the right direction.